saleor/graphql/account/resolvers.py

```diff
@@ -45,6 +47,19 @@ def resolve_staff_users(info, query):
     return gql_optimizer.query(qs, info)
 
 
+def resolve_user(info, id):
+    api_account = info.context.user or info.context.service_account
+    if api_account:
```

Comment:
`api_account` seems confusing? Why `api` and not simply `current_user`, `requester` or something of this kind?